src/SlashCommands.tsx

```diff
@@ -729,9 +729,9 @@ export const Commands = [
                         const cli = MatrixClientPeg.get();
                         const room = cli.getRoom(roomId);
                         if (!room) return reject(_t("Command failed"));
-
+                        const member = room.getMember(args);
+                        if (!member || member.membership !== "join") return reject(_t("Could not find user in room"));
```

Comment:
this probably needs a product decision as it addresses https://github.com/vector-im/riot-web/issues/12527